### ISSUE TITLE
Error loading xmltv guide

### ISSUE THREAD

**Issue by ivars777** [jellyfin/jellyfin#1703]
**Describe the bug**
Constantly showing error while loading xmltv guide for live tv.
Tried with 10.3.6, 10.3.7, nightly_20190831 (both windows and linux).
xmltv source - http://epg.blue.lv/ltc.xml

**Logs**

```
> [12:52:43] [INF] xmltv path: http://epg.blue.lv/ltc.xml
> [12:52:43] [ERR] Error adding metadata
> System.Xml.XmlException: '▼', hexadecimal value 0x1F, is an invalid character. Line 1, position 1.
>    at System.Xml.XmlTextReaderImpl.Throw(Exception e)
>    at System.Xml.XmlTextReaderImpl.Throw(String res, String[] args)
>    at System.Xml.XmlTextReaderImpl.ParseRootLevelWhitespace()
>    at System.Xml.XmlTextReaderImpl.ParseDocumentContent()
>    at System.Xml.XmlTextReaderImpl.Read()
>    at System.Xml.XmlReader.ReadToDescendant(String name)
>    at Emby.XmlTv.Classes.XmlTvReader.GetChannels()
>    at Emby.Server.Implementations.LiveTv.Listings.XmlTvListingsProvider.GetChannels(ListingsProviderInfo info, CancellationToken cancellationToken)
>    at Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV.GetEpgChannels(IListingsProvider provider, ListingsProviderInfo info, Boolean enableCache, CancellationToken cancellationToken)
>    at Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV.AddMetadata(IListingsProvider provider, ListingsProviderInfo info, List`1 tunerChannels, Boolean enableCache, CancellationToken cancellationToken)
>    at Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV.GetChannelsAsync(Boolean enableCache, CancellationToken cancellationToken)
> [12:52:43] [INF] Refreshing guide with 7 days of guide data
> [12:52:43] [INF] xmltv path: http://epg.blue.lv/ltc.xml
> [12:52:43] [ERR] Error adding metadata
> System.Xml.XmlException: '▼', hexadecimal value 0x1F, is an invalid character. Line 1, position 1.
>    at System.Xml.XmlTextReaderImpl.Throw(Exception e)
>    at System.Xml.XmlTextReaderImpl.Throw(String res, String[] args)
>    at System.Xml.XmlTextReaderImpl.ParseRootLevelWhitespace()
>    at System.Xml.XmlTextReaderImpl.ParseDocumentContent()
>    at System.Xml.XmlTextReaderImpl.Read()
>    at System.Xml.XmlReader.ReadToDescendant(String name)
>    at Emby.XmlTv.Classes.XmlTvReader.GetChannels()
>    at Emby.Server.Implementations.LiveTv.Listings.XmlTvListingsProvider.GetChannels(ListingsProviderInfo info, CancellationToken cancellationToken)
>    at Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV.GetEpgChannels(IListingsProvider provider, ListingsProviderInfo info, Boolean enableCache, CancellationToken cancellationToken)
>    at Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV.AddMetadata(IListingsProvider provider, ListingsProviderInfo info, List`1 tunerChannels, Boolean enableCache, CancellationToken cancellationToken)
>    at Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV.GetChannelsAsync(Boolean enableCache, CancellationToken cancellationToken)
> [12:52:43] [INF] xmltv path: http://epg.blue.lv/ltc.xml
> [12:52:43] [ERR] Error getting programs for channel LTV1
> 
```

**System (please complete the following information):**
 - OS:  Windows 10, Docker/Linux
 - Browser: N/A
 - Jellyfin Version: 10.3.6, 10.3.7, nighly_
 - Reverse proxy: caddy, no proxy

**Comment by 97carmine** [comment]
I reported that bug in version 10.3.6, but it was fixed in version 10.3.7. #1525

**Comment by Artiume** [comment]
Loaded the xml and found no issues. This can be resolved.
